src/router/App.js

```diff
@@ -17,12 +18,12 @@ const App = () => {
         <title>{t('webtitle')}</title>
       </Helmet>
       <Switch>
-        <Route path='/' exact component={LandingPage} />
-        <Route path='/about' exact component={About} />
-        <Route path='/contact' exact component={Contact} />
-        <Route path='/' exact component={LandingPage} />
-        <Route path='/forgotpassword' exact component={Forgot} />
-        <Route path='/reset' exact component={Reset} />
+        <Route path="/" exact component={LandingPage} />
+        <Route path="/about" exact component={About} />
+        <Route path="/contact" exact component={Contact} />
+        <Route path="/forgotpassword" exact component={Forgot} />
+        <Route path="/reset" exact component={Reset} />
+        <Route path="/signin" component={SigninPage} />
```

Comment:
These changes were not necessary to make. Check your prettier formatter and configure it to use single quote instead

Reply:
let me fix it, so far i think my prettier is configured with single quote